src/readers/src/TRM.cpp

```diff
@@ -4,12 +4,14 @@
 #include <shapes.h>
 
 #include <ranges>
+#include <cassert>
 
 namespace {
-size_t getNewId() {
+ShapeIdPair getNewId() {
 	static size_t id{0};
 	++id;
-	return id;
+	// TODO: pass disign_id
+	return {0, id};
```

Comment:
We don't need them here AFAIK, we create it in other place

Reply:
For future PR